### PR TITLE
Don't pass to DiffEqBase.ODEProblem invalid keyword check_length

### PR DESCRIPTION
**Pull request by isaacsas**
Fixes https://github.com/SciML/ModelingToolkit.jl/issues/1583

If this is ok, would be good to get a bug fix release once merged.

### CODE REVIEW

File: src/systems/diffeqs/abstractodesystem.jl [hunk 1]
```diff
@@ -774,10 +776,10 @@ numerical enhancements.
 struct ODEProblemExpr{iip} end
 
 function ODEProblemExpr{iip}(sys::AbstractODESystem,u0map,tspan,
-                             parammap=DiffEqBase.NullParameters();
+                             parammap=DiffEqBase.NullParameters(); check_length=true,
```
Comment (YingboMa) [inline src/systems/diffeqs/abstractodesystem.jl:779]: Trailing whitespace.

**Review by YingboMa — COMMENTED** [review]
There are quite a few trailing whitespaces.

**Comment by isaacsas** [comment]
I had turned that off as I found vscode was having frequent pauses with it on unfortunately...

**Comment by isaacsas** [comment]
@YingboMa ok to make a new patch release?

**Pull request merged**